volumes/csi-wrapper/hack/azure/nginx-kata-with-my-pvc-and-csi-wrapper.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-pv
+  labels:
+    app: nginx
+  namespace: default
```

Comment:
the namespace should same as the PVC one:
```
  namespace: kube-system
```

Reply:
Better to change the PVC namespace to `default`

Reply:
true, it's not sufficient to move the pod. But I do think we probably shouldn't pollute the `kube-system` ns w/ the demo.

Reply:
Sorry, somehow I forgot to push some changes like the ns change of the PVC. I'll test it first because I think we were depending on some serviceaccount.

Reply:
For `ibm-vpc-block-csi-driver` we use `kube-system` as the origin `ibm-vpc-block-csi-driver` are deployed to `kube-system`, and one secret from `kube-system` is needed.

If the azurefiles-csi-driver is name space unrelated, use `default` namespace is better.

Reply:
So I got it running in the `default` namespace by creating a new service account with the following ClusterRole:

```yaml
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: azure-files-csi-wrapper-podvm
rules:
  - apiGroups: ["confidentialcontainers.org"]
    resources: ["peerpodvolumes"]
    verbs: ["get", "list", "watch", "create", "delete", "update", "patch"]
  - apiGroups: ["confidentialcontainers.org"]
    resources: ["peerpodvolumes/status"]
    verbs: ["update"]
```

and Role:

```yaml
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: azure-files-csi-wrapper-podvm
  namespace: default
rules:
  - apiGroups: [""]
    resources: ["secrets"]
    verbs: ["get", "list", "watch"]
```